charts/sentence-collector/values.yaml

```diff
@@ -0,0 +1,60 @@
+replicaCount: 1
+
+image:
+  repository: mozilla/sentencecollector
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "2.0.9"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: "sentence-collector"
+
+externalSecret:
+  create: false
+
+serviceAccount:
+  create: true
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext:
+  fsGroup: 2000
+
+securityContext:
+  runAsNonRoot: true
+  runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: false
+  annotations: {}
+  hosts:
+    - host: chart-example.local
+      paths: []
+
+resources:
+  limits:
+    cpu: 500m
+    memory: 512Mi
```

Comment:
Knowing little about the application, I just want to confirm that 0.5 GB is an acceptable amount of memory for this?

Reply:
I'm not sure either will be enough to handle traffic, but I believe traffic will be fairly low. The limit is set because the chart runs  with the default values in a k8s cluster inside Travis for being tested, so we can't go big here :)
My plan for prod is enable "autoscaling" set it to max 5-7 replicas and probably bump the limit to 1Gb. But this is definitely something I have to measure once the app is deployed and gets some traffic.